src/main/java/de/danielbechler/diff/inclusion/InclusionService.java

```diff
@@ -24,13 +24,14 @@
 
 import java.util.Collection;
 import java.util.LinkedList;
+import java.util.Set;
 
 import static de.danielbechler.diff.inclusion.Inclusion.DEFAULT;
 import static de.danielbechler.diff.inclusion.Inclusion.EXCLUDED;
 import static de.danielbechler.diff.inclusion.Inclusion.INCLUDED;
 
 @SuppressWarnings("OverlyComplexAnonymousInnerClass")
-public class InclusionService implements InclusionConfigurer, IsIgnoredResolver
+public class InclusionService implements InclusionConfigurer, IsIgnoredResolver, CategoryResolver
```

Comment:
There is no need to add the `CategoryResolver` interface here, because that's the purpose of the `CategoryService`.